scripts/list-recent-buckets.sh

```
@@ -71,7 +71,7 @@ deletables=()
 for bucket in $buckets; do
     maybe_echo
     maybe_echo "Fetching metadata for ${bucket}..."
-    metadata="$(aws s3 cp "s3://${bucket}/metadata.json" - || echo '')"
+    metadata="$(aws s3 cp "s3://${bucket}/metadata.json" 2>/dev/null - || echo '')"
```

Suppresses error messages from output (to keep the output from this one clean).